spec/Powertrain/FuelSystem.vspec

```diff
@@ -40,6 +40,12 @@ TankCapacity:
   unit: l
   description: Capacity of the fuel tank in liters.
 
+TankNetCapacity:
+  datatype: float
+  type: sensor
+  unit: l
+  description: Net capacity of fuel tank in liters.
```

Comment:
What is the difference between "capacity" and "net capacity"? How come that it varies? I think a comment explaining difference would be good.

Reply:
I think i did here mistake with naming.